### DISCUSSION TITLE
Can I save the whole React Flow Diagram even the viewport only has the part of the diagram?

### DISCUSSION THREAD

**Discussion by iam-frankqiu** [xyflow/xyflow#4864] [Q&A]
If only part of the diagram was rendered, Can I save the whole diagram as a image?

**Comment by iam-frankqiu** [comment]
@moklick @peterkogo

**Comment by peterkogo** [comment]
https://reactflow.dev/examples/misc/download-image

**Reply by iam-frankqiu** [reply]
> https://reactflow.dev/examples/misc/download-image

I know this example. But my question is Can I download the whole diagram even the viewport has the part of the diagram.Thx

**Comment by iam-frankqiu** [comment]
I know this example. But my question is Can I download the whole diagram even the viewport has the part of the diagram.Thx
